api/v1alpha1/api.go

```diff
@@ -378,3 +429,36 @@ type AWSOIDCExchangeToken struct {
 	// which maps to the temporary AWS security credentials exchanged using the authentication token issued by OIDC provider.
 	AwsRoleArn string `json:"awsRoleArn"`
 }
+
+// RequestCost configures the request cost.
+type RequestCost struct {
+	// Type specifies the type of the request cost. The default is "OutputToken",
+	// and it uses "output token" as the cost. The other types are "InputToken" and "TotalToken".
+	//
+	// +kubebuilder:validation:Enum=OutputToken;InputToken;TotalToken
+	Type RequestCostType `json:"type"`
```

Comment:
```suggestion
	Type RequestTokenCostType `json:"type"`
```

Reply:
so you want me to change RequestCost as well to RequestTokenCost? feels a bit redundant though, works for me